components.css

```diff
@@ -1,9 +1,22 @@
-@import url("./CSS/navbar.css");
-@import url("./CSS/login.css");
+@import url("./components/navbar.css");
+@import url("./pages/login/login.css");
+@import url("./pages/products/products.css");
+@import url("./components/filters.css");
 
 :root{
     --lightergrey: #f8f8f8;
 }
 body{
     margin: 0;
+}
+.item-img{
+    width: 250px;
+    height: 300px;
+}
+.container-card{
+    width: 250px;
+    background: var(--white) !important;
```

Comment:
Try not to use important

Reply:
I had to use that to over-write the styles coming from my component library style sheet. just to change background color here.

Reply:
ok then it alright